tox.ini

```diff
@@ -1,10 +1,11 @@
 [tox]
-envlist = py39-{base,full}, py310-{base,full}, docs, lint
+envlist = py310-{base,full}, py311-{base,full}, py312-{base,full}, docs, lint
 
 [gh-actions]
 python =
-    3.9: py39-base, py39-full
-    3.10: py310-base, py310-full, lint, docs
+    3.10: py310
```

Comment:
`tox-gh-actions` should run all envs of a given prefix:
https://github.com/ymyzk/tox-gh-actions#factor-conditional-settings-python-version